src/index.css

```diff
@@ -316,3 +327,7 @@ input[type='number']::-webkit-outer-spin-button {
 .widget-ratings {
   display: flex !important;
 }
+
+.ReactModal__Overlay {
```

Comment:
Please move this style together with previous mentioned to a separate file: common/components/DateRangeSelector/DateRangeSelector.css

Reply:
fixed